src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -32,9 +33,17 @@
     private $createCacheItem;
     private $mergeByLifetime;
 
+    /**
+     * @var int|null The maximum length to enforce for identifiers or null when no limit applies
+     */
+    protected $maxIdLength;
+
     protected function __construct($namespace = '', $defaultLifetime = 0)
     {
-        $this->namespace = '' === $namespace ? '' : $this->getId($namespace);
+        $this->namespace = '' === $namespace ? '' : $this->getId($namespace).':';
```

Comment:
`:` is a reserved char per psr6 so we use it internally to be free from any collision when concatenating keys and namespace later on